frontera/contrib/backends/cassandra/__init__.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import absolute_import
+from cassandra.cluster import Cluster
+from cassandra.cqlengine import connection
+from cassandra.query import dict_factory
+from cassandra.cqlengine.management import sync_table
+from cassandra.cqlengine.management import drop_table
+from frontera.core.components import DistributedBackend
+from frontera.contrib.backends import CommonBackend
+from frontera.contrib.backends.cassandra.components import Metadata, Queue, States
+from frontera.utils.misc import load_object
+
+
+class CassandraBackend(CommonBackend):
+    def __init__(self, manager):
+        self.manager = manager
+        settings = manager.settings
+        cluster_ips = settings.get('CASSANDRABACKEND_CLUSTER_IPS')      # Format: ['192.168.0.1', '192.168.0.2']
```

Comment:
It would be great to include these settings in documentation and move format description there. I know docs are boring, but there is nothing we can do.